index.html

```diff
@@ -14,45 +14,254 @@
+            <div></div>
+            <p>Back End Dev</p>
+            <div></div>
+            <p>2015</p>
+          </div>
+          <p class="service">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="projectBTN"><button>See Project</button></div>
+        </article>
+
+        <article>
+          <img
+            class="card_image"
+            src="Snapshoot Portfolio2.png"
+            alt="project2 image"
+          />
+          <h3 class="projectName_holder">Multi-Post Stories</h3>
+          <div class="canopy">
+            <h4>CANOPY</h4>
+            <div></div>
+            <p>Back End Dev</p>
+            <div></div>
+            <p>2015</p>
+          </div>
+          <p class="service">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="projectBTN"><button>See Project</button></div>
+        </article>
+
+        <article>
+          <img
+            class="card_image"
+            src="Snapshoot Portfolio3.png"
+            alt="project3 image"
+          />
+          <h3 class="projectName_holder">Tonic</h3>
+          <div class="canopy">
+            <h4>CANOPY</h4>
+            <div></div>
+            <p>Back End Dev</p>
+            <div></div>
+            <p>2015</p>
+          </div>
+          <p class="service">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="projectBTN"><button>See Project</button></div>
+        </article>
+
+        <article>
+          <img
+            class="card_image"
+            src="Snapshoot Portfolio4.png"
+            alt="project4 image"
+          />
+          <h3 class="projectName_holder">Multi-Post Stories</h3>
+          <div class="canopy">
+            <h4>CANOPY</h4>
+            <div></div>
+            <p>Back End Dev</p>
+            <div></div>
+            <p>2015</p>
+          </div>
+          <p class="service">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="projectBTN"><button>See Project</button></div>
+        </article>
+      </section>
+
+      <section class="about">
+        <h2>About<br />Myself</h2>
```

Comment:
- [ ] please, like in the original design, the about myself heading is on a single line. Kindly fix that
- [ ] you can the languages and techs are centralized, but yours is aligned to the left. Please fix that. 
yours
![Screenshot (1220)_LI](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/65298048/f4eb842a-a62d-400a-8733-3116afe6d8fe)
figma
![Screenshot (1219)](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/65298048/1439c2ab-1194-4c5b-b3f2-ea8b408fed4a)